plugins/inputs/etherscan/balance.go

```diff
@@ -0,0 +1,282 @@
+package etherscan
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"math/big"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	ErrBalanceAddressCountExhausted = errors.New("maximum address limit reached for batch request")
+	ErrReturnCodeNotOK              = errors.New("return code was not OK")
+	ErrNotValidEthAddress           = errors.New("invalid Ethereum address")
+)
```

Comment:
Maybe it would be better to define a custom error (see e.g. https://gobyexample.com/errors) with an error number instead!?